File: server/src/main/java/com/example/server/user/dto/ExerciseDto.java
package com.example.server.user.dto;

import com.example.server.user.model.User;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter @Setter
@NoArgsConstructor
public class ExerciseDto {
    private Long id;
    private String name;
    private Long height;
    private Long weight;
    private Long walk;
    private Long goal;
    private String gender;
    private int totalRanking;
    private int majorRanking;

    public ExerciseDto(User user) {
        this.id = user.getId();
        this.name = user.getName();
        this.height = user.getHeight();
        this.weight = user.getWeight();
        this.walk = user.getWalk();
        this.goal = user.getGoal();
        this.gender = user.getGender();
    }
}

File: app/src/main/java/com/example/healthycollege/data/model/ExerciseDTO.kt
package com.example.healthycollege.data.model

data class ExerciseDTO (
    var id : Long,
    var name : String,
    var height : Long,
    var weight : Long,
    var walk : Long,
    var goal : Long,
    var gender : String,
    var totalRanking : Int,
    var majorRanking : Int
)